backoffice/src/main/scala/TravisTest.scala

```diff
@@ -0,0 +1,22 @@
+import com.fasterxml.jackson.databind.ObjectMapper
+
```

Comment:
It's just a dummy class to let git track the scala folder, we can remove it in the next task of the backoffice, but at least I'll rename it.